src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx

```diff
@@ -15,22 +20,70 @@ import TokenAvatar from '../TokenAvatar/index.ts';
 
 import { type BalanceTableFieldModel } from './types.ts';
 
+export const useBalancesData = (): BalanceTableFieldModel[] => {
+  const {
+    colony: { tokens: colonyTokens, balances },
+  } = useColonyContext();
+  const selectedDomain = useGetSelectedDomainFilter();
+
+  const tokensData = useMemo(
+    () =>
+      colonyTokens?.items.map((item) => {
+        const currentTokenBalance =
+          getBalanceForTokenAndDomain(
+            balances,
+            item?.token?.tokenAddress || '',
+            selectedDomain ? Number(selectedDomain.nativeId) : undefined,
+          ) || 0;
+        const decimals = getTokenDecimalsWithFallback(item?.token.decimals);
+        const convertedValue = convertToDecimal(
+          currentTokenBalance,
+          decimals || 0,
+        );
+
+        const formattedValue = getFormattedNumeralValue(
+          convertedValue,
+          currentTokenBalance,
+        );
+
+        return {
+          ...item,
+          balance: typeof formattedValue === 'string' ? formattedValue : '',
+        };
+      }),
+    [colonyTokens, balances, selectedDomain],
+  );
+
+  const sortedTokens =
+    useMemo(
+      () =>
+        tokensData?.sort((a, b) => {
+          if (!a.balance || !b.balance) return 0;
+          return parseInt(b.balance, 10) - parseInt(a.balance, 10);
+        }),
+      [tokensData],
+    ) || [];
+
+  return sortedTokens;
+};
+
 export const useBalanceTableColumns = (
   nativeToken,
   balances,
   nativeTokenStatus,
   domainId = 1,
 ): ColumnDef<BalanceTableFieldModel, string>[] => {
-  const columnHelper = useMemo(
-    () => createColumnHelper<BalanceTableFieldModel>(),
-    [],
-  );
+  const isMobile = useMobile();
+
+  const columns: ColumnDef<BalanceTableFieldModel, string>[] = useMemo(() => {
+    const columnHelper = createColumnHelper<BalanceTableFieldModel>();
 
-  const columns: ColumnDef<BalanceTableFieldModel, string>[] = useMemo(
-    () => [
+    return [
       columnHelper.display({
         id: 'asset',
         header: () => formatText({ id: 'table.row.asset' }),
+        headCellClassName: isMobile ? 'pr-2' : undefined,
+        cellContentWrapperClassName: isMobile ? 'pr-2' : undefined,
```

Comment:
would it be possible to put these classNames inside of the `cell` function? so we render a span with  this className or something

Reply:
I can't find anything in the docs. I used a function that we already have implemented to update the spacings on the wrapper.

Reply:
can we try it out? it's probably not a huge thing, just add the className to the `cell` attribute like we already do in some cases
![image](https://github.com/JoinColony/colonyCDapp/assets/23449297/8a05ed89-531f-4ca8-bc36-adb5e750a6fb)

Reply:
But this is an additional element inside the cell's wrapper, so it won't work the way I need it to work